.github/workflows/pythonpackage.yml

```diff
@@ -20,23 +19,14 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install pipenv
-        pipenv install --dev
-    - name: Lint with flake8
+        pip install poetry
```

Comment:
nit it would be good to install a specific poetry version since there can be changes between versions in the lock files.

There's also a github action for installation that can be nice.